src/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<link rel="stylesheet" href="style.css">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-Do List</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+</head>
+
+<body>
+    <h4 class="intro">"Stay organized and boost your productivity with our intuitive and sleek to-do list web design. Manage your tasks effortlessly and achieve your goals efficiently."</h4>
+    <div class="container">
+
+        <div>
+            <ul class="for-title">
+                <li>
+                    <p class="to-do">Today's To Do</p>

```

Comment:
- [ ] As per given design, I don't think you should have a list with 2 bullets as per the screenshot below.
![image](https://github.com/Hsuwaddyhtat22/To-Do-List-mini-PJ/assets/62849872/dec02f9d-6fc2-4db5-873b-eb6808aa41a7)